### ISSUE TITLE
Dyn's record weighting has a range of 1-15

### ISSUE THREAD

**Issue by ross** [octodns/octodns#368]
Not sure how this was missed previously, but apparently dyn's range is 1-15 which doesn't line up well with dynamic's 0-255. We'll probably need to `int(weight/16)` which will work, but reduce the "resolution" quiet drastically. Other option is to limit dynamic's range from 1-15 and then `int(weight*16)` for Route53. Will need to think about it.

```
dyn.tm.errors.DynectCreateError: rs_chain_0_record_set_0_record_4_weight: Too great. rs_chain_0_record_set_0_record_5_weight: Too great. rs_chain_0_record_set_0: A field contained data that was invalid. rs_chain_0_record_set_0_record_4: A field contained data that was invalid. rs_chain_0: A field contained data that was invalid. response_pool_add: The operation failed to complete successfully. rs_chain_0_record_set_0_record_5: A field contained data that was invalid. response_pool_add: could not create response pool.
 ** [timing :: dns-extauth-ccc76ca.dc2-iad.github.net] command finished in 99.01s
```

**Comment by ross** [comment]
This was resolved long ago.

**Comment by viranch** [comment]
For posterity: fixed by #792
